### ISSUE TITLE
No errors or tables created

### ISSUE THREAD

**Issue by kevinsimper** [elwinar/rambler#21]
I made a migration file and a config file, it did create the migration table in the database, but when I run `apply` nothing happens and no errors are outputted.

It would be cool if there were some output to debug what happens :)

**Comment by elwinar** [comment]
That's a good idea, I'm still not decided about that output should be. 
What would you like ?

In the meantime, if the migration table is created then the `directory` 
configuration option probably points to a directory with no sql file, so 
no migration are detected, so no migrations are applied.

R;

**Comment by kevinsimper** [comment]
The problem is that I have sql files with -- rambler up --rambler down and it does *not* run them 🤔

I don't know if it is because it looks at the wrong directory and did not find any or it broke.

**Comment by elwinar** [comment]
It's probably your path that is wrong, as there are logs in case of error.

I'm thinking about the logs that could be useful. What do you think about:

- Created the migration table
- Applied a migration
- Reversed a migration
- Errors

See anything else ?

**Comment by kevinsimper** [comment]
No migrations found, maybe? :)

**Comment by elwinar** [comment]
A count of both the available and applied migrations is a good idea too.
I'll start working on it soon.

**Comment by elwinar** [comment]
Didin't had the time this week-end… I'll try to manage one or two hours 
this week.

**Comment by elwinar** [comment]
Found a couple of hours today (was about time!).

I wrote a logger for the project, based on an article by Dave Cheney about logging. I'll take some time to tweak it to what I want, and may at some point release it independently from rambler itself, but not today.

The application have a new flag `--debug` for debug messages(hopefully useful for debugging configuration errors), and basic informational messages are always displayed. Errors will be on `stderr`

I'm thinking about using logfmt-style logs, are these are better for automated consumption, but I believe that these cases are rare, and even then the logs of rambler aren't exactly actionable, so an end-user focused display may be the best option.

If you want to checkout the work-in-progress, it is in the [logs](https://github.com/elwinar/rambler/tree/logs) branch.

Cheers,
R;
